.tekton/copy-subscription-manager-activation-key-oci-ta.yaml

```
@@ -0,0 +1,55 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: copy-subscription-manager-activation-key-oci-ta
+  namespace: rh-acs-tenant
+# TODO(ROX-23812): Refactor to a task bundle
+spec:
+  description: Smuggles an activation key next to the application source code that enables access to RHEL RPMs.
+  params:
+  - name: SOURCE_ARTIFACT
+    description: The Trusted Artifact URI pointing to the application source code with
+      the additional smuggled activation key. This should be the result of the git-clone task,
+      results from other tasks might fail as dirty.
+    type: string
+  - name: ociStorage
+    description: The OCI repository where the Trusted Artifacts are stored.
+    type: string
+  - name: ociArtifactExpiresAfter
+    description: Expiration date for the trusted artifacts created in the
+      OCI repository. An empty string means the artifacts do not expire.
+    type: string
+    default: ""
```

Well, we probably want artifacts to expire. I suggest removing the default value which would force the caller to provide the parameter value because otherwise the pipeline fails.
```suggestion
  - name: ociArtifactExpiresAfter
    description: Expiration date for the trusted artifacts created in the
      OCI repository.
    type: string
```